### PR TITLE
Create murzynek-cocoa-cake.md

### PR DESCRIPTION
**Pull request by pkbladesmith**
first time adding something to github

### CODE REVIEW

File: src/murzynek-cocoa-cake.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- ⏲ Prep time: 15 min
+- 🍳 Cook time: 40-50 min
+- 🍽 Servings: depends
+
+## Ingredients
+- 2 cups flour
+- 0.5 cup of sugar
+- 1 tsp. baking soda (sodium bicarbonate)
+- 2 sp. cocoa powder
```
Comment (hallzy) [inline src/murzynek-cocoa-cake.md:16]:
```suggestion
- 2 tbsp. cocoa powder
```
do you mean table spoon?

File: src/murzynek-cocoa-cake.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- 2 whole eggs
+- 100 grams of butter
+- 1 cup fat milk
+- 3 sp. high quality jam of choice
```
Comment (hallzy) [inline src/murzynek-cocoa-cake.md:20]:
```suggestion
- 3 tbsp. high quality jam of choice
```
table spoon?

File: src/murzynek-cocoa-cake.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Directions
+- Heat the oven to 180 C/356 F
```
Comment (hallzy) [inline src/murzynek-cocoa-cake.md:23]:
```suggestion
- Heat the oven to 180°C/356°F
```

File: src/murzynek-cocoa-cake.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Mix wet ingredients
+- Add dry ingredients
+- Mix until the consistence is nice and smooth
```
Comment (hallzy) [inline src/murzynek-cocoa-cake.md:26]:
```suggestion
- Mix until the consistency is nice and smooth
```

**Review by hallzy — CHANGES_REQUESTED** [review]

File: src/murzynek-cocoa-cake.md (hunk 2 above)
Comment (pkbladesmith) [reply src/murzynek-cocoa-cake.md]: yes, table spoon

File: src/murzynek-cocoa-cake.md (hunk 4 above)
Comment (pkbladesmith) [reply src/murzynek-cocoa-cake.md]: How do I add pic to this pull request? Kinda new to github.

**Comment by hallzy** [comment]
@pkbladesmith I use the git command tool so I am not 100% sure how you do it from github, but this would be my best guess...

Go to your fork of based.cooking [here](https://github.com/pkbladesmith/based.cooking)

There is a button that probably says "master", if you click it is becomes a dropdown. You have to select the branch that you created for this PR. That is `patch-1` in this case (github probably created this branch for you if you are confused).

Once you have selected `patch-1` now you are looking at your fork that you have referenced in this PR.

Click the 'Add File' button and do the same as before when you added your image last time.

Basically, your PR with the image added the image to your `master` branch, but you have to add it to your `patch-1` branch because that is the branch you are using for this pull request...

Not the most intuitive thing for beginners I am sure...

**Review by hallzy — APPROVED** [review]

**Comment by hallzy** [comment]
Looks like you figured it out :D

**Comment by pkbladesmith** [comment]
How do I make this patch merge with the main repo?
